### ISSUE TITLE
bug: cloned title element doesn't reflect color and font-size

### ISSUE THREAD

**Issue by EinfachHans** [ionic-team/ionic-framework#26218]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x
- [ ] Nightly

### Current Behavior

When using a `--color` or `font-size` on an `<ion-title>` on an collapsable header, the cloned title element that is used for the animation doesn't reflect this styles, what results in a buggy animation:

![ezgif com-gif-maker](https://user-images.githubusercontent.com/20899975/199695843-375f9556-3225-42ab-a191-1122673ea075.gif)

The cloned element always is black and jumps after the animation is finished into the original state. Also i mentioned that the BackButton Text is not correctly aligned to the cloned element (it should be over each other to have the effect of turning into the title)

### Expected Behavior

- The cloned element should respect the original title's styling
- The animation should match with the back button text

### Steps to Reproduce

See attached repo and the gif above

### Code Reproduction URL

https://github.com/EinfachHans/ionic-title-animation-bug

### Ionic Info

Ionic:

   Ionic CLI                     : 6.20.3 (/Users/hans/.nvm/versions/node/v16.17.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 6.3.4
   @angular-devkit/build-angular : 14.2.8
   @angular-devkit/schematics    : 14.2.8
   @angular/cli                  : 14.2.8
   @ionic/angular-toolkit        : 6.1.0

Capacitor:

   Capacitor CLI      : 4.4.0
   @capacitor/android : not installed
   @capacitor/core    : 4.4.0
   @capacitor/ios     : not installed

Utility:

   cordova-res (update available: 0.15.4) : 0.15.1
   native-run                             : 1.7.1

System:

   NodeJS : v16.17.0 (/Users/hans/.nvm/versions/node/v16.17.0/bin/node)
   npm    : 8.15.0
   OS     : macOS

**Comment by liamdebeasi** [comment]
Thanks for the report. The large title should be styled globally as opposed to within the context of a particular page in order for the styles to apply to the cloned element during transition. This is explained in the [Theming](https://ionicframework.com/docs/api/title#theming) documentation for `ion-title`:

> When styling the text color of the large title, you should target the large title globally as opposed to within the context of a particular page or tab, otherwise its styles will not be applied during the navigation animation.

**Comment by EinfachHans** [comment]
@liamdebeasi Okay thank you - i didn't searched there to be honest. 😅 But this only works if every large title is styled the same right?

**Comment by liamdebeasi** [comment]
Yes, it is recommended to have a uniform theme throughout the application so we recommend styling the large title consistently too.

**Comment by EinfachHans** [comment]
Got it, works for us in this case because we have a uniform theme, but i could imagine having a different background color on a page for example which could result in having a different font color only for that page as well 🤔

**Comment by liamdebeasi** [comment]
If someone has that use case, I'd definitely encourage them to open a feature request and we'll be happy to consider it.

**Comment by EinfachHans** [comment]
Alright, thanks 👍🏼
